### DISCUSSION TITLE
Add category of discussion to notifications page

### DISCUSSION THREAD

**Discussion by huehnerlady** [community/community#3987] [Discussions]
Hi,

many users use the category to not include the context in the title anymore - which I would say makes sense.
The problem is that in the [notifications page](https://github.com/notifications?query=is%3Aunread) does not show the categories in the overview, so some titles seem out of context until you click into them.

![image](https://user-images.githubusercontent.com/12533979/118936854-35baff80-b94d-11eb-9ec2-224491965407.png)

So it would be great if you could add the category into the notifications page. 
If this is a generic structure maybe you could add the category into the title.
Example for the highlighted notification would be:

`Discussions Feedback: Archive/Close a discussion`

bonus would be that you could sort via repository and then afterwards group by category.

**Comment by PranamBhat** [comment]
This would create more problems. Since we already have categorized Mobile, Code Spaces, Discussions and Sponsors, this would not make a sense out of it. Thanks for the suggestion.

**Reply by huehnerlady** [reply]
With my suggestion I did not meant for it to be the actual title, but just come up in the notifications menu. Also happy to have it another way (eg. column/grouping/label)

It is quite frustrating to just see "Translation for git projects" in the notifications and not being able to know which category it belongs to. I always have to go in and see if it interests me.

Maybe that would be less frustrating, if you could actually filter which categories you wanna follow, but still would say that this is something that belongs in the notifications overview.
